### ISSUE TITLE
npm start not working

### ISSUE THREAD

**Issue by Harshit671** [near-apps/payments-contract#3]
When I use the command npm start it's not running the app

**Comment by mattlockyer** [comment]
A little more information would help.

I use yarn mostly so:
npm i -g yarn
yarn
yarn start

**Comment by Harshit671** [comment]
Yes , I clone the repo and run npm install . After that I ran npm start and it was not working

**Comment by mattlockyer** [comment]
Try yarn. Should be working.

Can you provide errors?
